jupyterlab_latex/config.py

```diff
@@ -15,7 +15,7 @@ class LatexConfig(Configurable):
     bib_command = Unicode('bibtex', config=True,
         help='The BibTeX command to use when compiling ".tex" files.' +\
              'Only used if disable_bibtex is not set to True')
-    synctex_command = Bool('synctex', config=True,
+    synctex_command = Unicode('synctex', config=True,
         help='Whether to use the synctex command when syncronizing between .tex and .pdf files.')
```

Comment:
Should it also be changed to

```suggestion
        help='The synctex command to use when syncronizing between .tex and .pdf files.')
```

Reply:
Thanks @krassowski for your your review and sorry for the late reply!